Makefile

```diff
@@ -12,9 +12,12 @@ compile:
 doc:
 	$(REBAR) doc $(REBAR_FLAGS)
 
-test: compile
+test: compile ct
```

Comment:
If `ct` target already calls `compile` (below at line #18) test could be just:
`test: ct`

The way it is now, will run `compile` 2 times when calling `test`.